homeassistant/components/config/config_entries.py

```diff
@@ -7,7 +7,7 @@
     FlowManagerIndexView, FlowManagerResourceView)
 
 
-REQUIREMENTS = ['voluptuous-serialize==1']
+REQUIREMENTS = ['voluptuous-serialize==2']
```

Comment:
The release should be `2.0.0`.

Reply:
Just thought @balloob may have reason use major version at first place.